README.md

```diff
@@ -1242,6 +1244,125 @@ spec:
     emphemeral: true # VERY important. otherwise data inside the workdir and /tmp is not cleared between builds
 ```
 
+#### Docker image layers caching
+
+> **Note**: Ensure that the volume mount is added to the container that is running the Docker daemon.
+
+`docker` stores pulled and built image layers in the [daemon's (note not client)](https://docs.docker.com/get-started/overview/#docker-architecture) [local storage area](https://docs.docker.com/storage/storagedriver/#sharing-promotes-smaller-images) which is usually at `/var/lib/docker`.
+
+By leveraging RunnerSet's dynamic PV provisioning feature and your CSI driver, you can let ARC maintain a pool of PVs that are
+reused across runner pods to retain `/var/lib/docker`.
+
+_Be sure to add the volume mount to the container that is supposed to run the docker daemon._
+
+By default, ARC creates a sidecar container named `docker` within the runner pod for running the docker daemon. In that case,
+it's where you need the volume mount so that the manifest looks like:
+
+```yaml
+kind: RunnerSet
+metadata:
+  name: example
+spec:
+  template:
+    spec:
+      containers:
+      - name: docker
+        volumeMounts:
+        - name: var-lib-docker
+          mountPath: /var/lib/docker
```

Comment:
Would this also cache state or container volume? If this is only for layers, would `/var/lib/docker/overlay2` be enough?

Reply:
I'm not a docker expert, you might be correct but how can you confirm that's actually correct? To me, the roles of those subdirectories seems not explained in the official docker doc.

Also, I see some overlay2 related metadata reside in `/var/lib/docker/image/overlay2`. Can we safely omit that? 🤔

Reply:
I'm not very sure on this either, but I think we wouldn't want anything other the the layers persists through different runs. So `/var/lib/docker` seems a bit vague.

I have `canary` setup but haven't able to get PV persist over runs.

Reply:
I'm not very sure on this either, but I think we wouldn't want anything other the the layers persists through different runs. So `/var/lib/docker` seems a bit vague.

I have `canary` setup but haven't able to get PV persist over runs.

Reply:
@Gentoli You need several job runs to let the next runner pod reuse the existing PV, because ARC doesn't block the runner pod creation while other PVCs and PVs are being unbound(otherwise every runner pod creation will be delayed by previous runner pod's PV ubounding)

Reply:
> So /var/lib/docker seems a bit vague.

@Gentoli Yeah, but trying to rely on an undocumented thing doesn't sound like a good idea, especially if you're going to use it in production.

Reply:
> You need several job runs
Oh, when it gets to have to PV, caching with `/var/lib/docker` did work. `/var/lib/docker/overlay2` didn't.